src/app/app.component.spec.ts

```diff
@@ -279,7 +279,7 @@ describe('AppComponent', () => {
         let loadOmnibarCallback: Function;
 
         const runOutsideAngularSpy = spyOn(zone, 'runOutsideAngular').and.callFake(
-          (cb: Function) => {
+          function (cb: Function): any {
```

Comment:
Why do these have to change from fat-arrow syntax?  The return type could be added there as well, right?

Reply:
Reverted!